library.properties

```diff
@@ -1,11 +1,11 @@
 name=ChirpSDK
-version=3.4.0
+version=3.4.1
 author=Asio Ltd <developers@chirp.io>
 maintainer=Joe Todd <joe@chirp.io>
 sentence=Chirp SDK
 paragraph=Send and receive data over sound
 category=Communication
-url=https://developers.chirp.io
-architectures=cortex-m4,esp32,cortex-m0plus,stm32f4,samd,mbed
+url=https://developers.chirp.io/docs/getting-started/arduino
+architectures=cortex-m4,esp32,cortex-m0plus,stm32f4,samd,mbed,avr,mk64fx512
```

Comment:
We don't have support for AVR, is this wanted ?

Reply:
It's a similar idea for `samd` and `cortex-m0plus` I think, the IDE sometimes seems to print it as `avr` architecture so I thought it better to include both...

Reply:
Oh okay, well AVR micro controllers are not based on arm whilst samd are but if the ide throws some warnings that's better to keep it then